comms/core/src/protocol/messaging/extension.rs

```diff
@@ -39,9 +39,9 @@ use crate::{
     runtime::task,
 };
 
-/// Buffer size for inbound messages from _all_ peers. This should be large enough to buffer quite a few incoming
-/// messages before creating backpressure on peers speaking the messaging protocol.
-pub const INBOUND_MESSAGE_BUFFER_SIZE: usize = 100;
+/// Buffer size for inbound messages from _all_ peers. Is the message consumer is slow to get through this queue,
+/// sending peers will start to experience backpressure (this is a good thing).
+pub const INBOUND_MESSAGE_BUFFER_SIZE: usize = 10;
```

Comment:
Not quite sure about this change. It feels that at least this should be configurable as we might have different roles for base nodes requiring different buffer sizes.

Reply:
Otherwise LGTM

Reply:
In my tests I had this as 1, I made it 10 because there would be a small performance gain by having some buffering, however more than a small amount of buffering is pointless. There is no reason I can think of to make this user configurable, this is completely internal to the system and should just work.

Reply:
Perfect